src/styles/typography/index.md.njk

```diff
@@ -79,6 +79,10 @@ Use:
 - `govuk-!-text-align-right` to align text to the right
 - `govuk-!-text-align-centre` to align text to the centre
 
+You should usually left-align body copy that's read left to right. Right-aligned body copy can be hard for users to find and read when they magnify their screen.
+
+Do not 'justify' blocks of body copy so that they're aligned to both the left and right margins. Doing this creates blank spaces, which can also make the text hard to read.
```

Comment:
```suggestion
Do not 'justify' blocks of body copy so that they're aligned to both the left and right margins. Doing this creates wider spaces between words, which can also make the text hard to read.
```